sdk/servicebus/azure-servicebus/azure/servicebus/_common/tracing.py

```diff
@@ -104,7 +104,16 @@ def send_trace_context_manager(
     span_name: str = SPAN_NAME_SEND,
     links: Optional[List[Link]] = None
 ) -> Iterator[None]:
-    """Tracing for sending messages."""
+    """Tracing for sending messages.
+    :param sender: The sender that is sending the message.
+    :type sender: ~azure.servicebus.ServiceBusSender or ~azure.servicebus.aio.ServiceBusSenderAsync
+    :param span_name: The name of the tracing span.
+    :type span_name: str
+    :param links: A list of links to include in the tracing span.
+    :type links: list[Link] or None
```

Comment:
would we want to update these to?
```suggestion
    :type links: list[~azure.core.tracing.Link] or None
```